app/src/main/res/layout/card_dictionary.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView xmlns:tools="http://schemas.android.com/tools"
+    xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/card_view"
+    android:layout_width="match_parent"
+    android:layout_height="43dp"
+    android:layout_margin="10dp"
+    card_view:cardCornerRadius="10dp">
+
+    <ImageView
+        android:id="@+id/topic_title_background"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:src="@drawable/default_card_placeholder"
+        android:contentDescription="Card Background" />
+
+    <TextView
+        android:id="@+id/topic_title_name"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:gravity="center"
+        android:textColor="#fff"
+        android:textSize="24sp"
+        android:textAllCaps="true"
+        android:textStyle="bold"
+        tools:text="Holi" />
```

Comment:
I do not know your coding standards, but avoid hard-coding text like this. @atm1504 Please review this.

Reply:
@fliptrail we also don't hardcode strings. But here we are just making sample cards. Also "tools:text" is the data shown in our editor. This data don't gets displayed. This are for developers to see how it would look. So it won't effect.

Reply:
That's text under tools namespace, it only appears while we are designing our layout in android studio.

Reply:
Ok. Thank you!